helm/prometheus-rules/templates/alerting-rules/alertmanager.rules.yml

```diff
@@ -15,21 +15,45 @@ spec:
       annotations:
         description: '{{`AlertManager {{ $labels.integration }} notifications are failing.`}}'
         opsrecipe: alert-manager-notifications-failing/
-      expr: rate(alertmanager_notifications_failed_total{cluster_id!~"argali|giraffe", integration!~"slack|webhook"}[5m]) > 0
+      expr: rate(alertmanager_notifications_failed_total{cluster_id!~"akita|argali|giraffe", integration!="opsgenie"}[5m]) > 0
       for: 10m
       labels:
         area: empowerment
         severity: page
         team: atlas
         topic: monitoring
+        cancel_if_outside_working_hours: "true"
+    - alert: AlertManagerOpsgenieNotificationsFailing
+      annotations:
+        description: '{{`AlertManager {{ $labels.integration }} notifications are failing.`}}'
+        opsrecipe: alert-manager-notifications-failing/
+      expr: rate(alertmanager_notifications_failed_total{cluster_id!~"akita|argali|giraffe", integration="opsgenie"}[5m]) > 0
+      for: 2m
```

Comment:
I set to 2m because when an alert fails to notify opsgenie, it increases just by one `alertmanager_notifications_failed_total` counter. 
But the rate over 5m fails down to 0 in less than 5 minutes (~3 or 4 minutes).

Reply:
I'm no sure I follow 😅

Reply:
But in any case the for should bé thé same for both alerts